tests/xrt/100_ert_ncu/xrtxx-ip.cpp

```diff
@@ -253,6 +284,7 @@ run(int argc, char** argv)
   size_t secs = 0;
   size_t jobs = 1;
   size_t cus  = 1;
+  int32_t timeout = -1;
```

Comment:
It looks like this variable is only being set and not used.  

Is this the expected behavior?

Reply:
updated